README.md

```diff
@@ -78,7 +78,7 @@ For more see [examples folder](./examples).
 
 ### Parity-specific APIs
 - [ ] Parity read-only: `parity_*`
-- [ ] Parity accounts: `parity_*`
+- [x] Parity accounts: `parity_*` (Currently being implemented @CPerezz)
```

Comment:
Let's change this to mention that it's only partially implemented instead. Feel free to add yourself to `CONTRIBUTORS.md` file or sth like this.